Sources/Tropos/Views/CircularProgressLayer.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+@objc(TRCircularProgressLayer) class CircularProgressLayer: CALayer {
+    @objc var progress: CGFloat = CGFloat(0)
+    @objc var radius = CGFloat(15.0)
+    @objc let outerRingWidth = CGFloat(3.0)
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+    override init() {
+        super.init()
+        self.actions = [
+            "bounds": NSNull(),
+            "contents": NSNull(),
+            "position": NSNull(),
+        ]
+        self.contentsScale = UIScreen.main.scale
+        self.needsDisplayOnBoundsChange = true
+    }
+
+    init(layer: CircularProgressLayer) {
+        super.init(layer: layer)
+        progress = layer.progress
+        radius = layer.radius
+    }
+
+    override func draw(in ctx: CGContext) {
+        let center = CGPoint(x: self.bounds.width / 2.0, y: self.bounds.height / 2.0)
```

Comment:
I think that in order to match the previous behaviour of `TRCGPointMakeIntegral()`, these should be `round()`ed. Maybe a `CGPoint` extension with a func `integral()` that returns a new point with each dimension rounded?